woocommerce/woocommerce
# pnpm 7 update: Remove unused dashes

### All Submissions:






### Changes proposed in this Pull Request:



pnpm passes along [all arguments after the command to the underlying script](https://pnpm.io/cli/run). Previously  we were using `--` to tell pnpm to do that, but now isn't required.

If I'm understanding correctly, the `--` is still useful in cases where a script calls another script that is not pnpm in which case it instructs pnpm to pass any arguments along to the underlying script.

### How to test the changes in this Pull Request:

1. Test one of the command, ie `pnpm -r --filter=woocommerce/client/admin run test:watch`
2. See that it succeeds without error

### Other information:




### FOR PR REVIEWER ONLY:
